src/applications/personalization/profile/util/connected-apps.js

```diff
@@ -5,6 +5,7 @@ export const mockConnectedApps = [
     attributes: {
```

Comment:
Hey @erikphansen. Do you happen to know why we're using this mock data? I was able to call the endpoint locally so it might just be that [this comment](https://github.com/department-of-veterans-affairs/vets-website/blob/f07252f21161b7911ef6fc148212ce8a7c3206d7/src/applications/personalization/profile/components/connected-apps/actions/index.js#L27) is outdated. Happy to remove the mock in this PR if possible.

Reply:
@sanlouise might know about why we need to mock this locally.

Reply:
@LeKeve From what I remember, when I implemented the UI for the connected apps on the profile, I was unable to hit the endpoint locally and return any connected apps for local users. Do you actually get a non-empty payload back? I wonder how those apps could persist on a local user 🤔 

Another consideration is testability - even if you are able to hit the endpoint locally, if you want to check the adding/deleting functionality on the UI, you'd need to reconnect apps manually each time after having deleted them - which can be a bit of a pain and this process might need to be documented better.

I'n not opposed to keeping the mock data if you are not too opinionated?

Reply:
Ok, that makes sense. I used the [Claims Test OAuth App](https://va-claims-development.herokuapp.com/) to test locally but it's definitely easier to test using the mock. 

I agree and am not opposed to keeping the mock data. Thanks for answering.